rtsmith.cpp

```diff
@@ -41,18 +44,30 @@ int RtSmith::mainImpl(const CompilerResult &compilerResult) {
 
     auto &fuzzer = RtSmithTarget::getFuzzer(*programInfo);
 
+    std::filesystem::path currentDir = std::filesystem::current_path();
```

Comment:
These should all be options for the tool. The output directory, the name of the output file and whether you want to write to a file or to stdout. 
Please take a look at https://github.com/fruffy/flay/blob/master/options.cpp on examples how this could be done.